x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/text_expansion_search_sparse_vector.yml

```diff
@@ -0,0 +1,109 @@
+# This test uses the simple model defined in
```

Comment:
Added a separate YAML test that performs the same checks for `text_expansion` queries for `sparse_vector` field type